gulp/tasks/docs.js

```diff
@@ -29,12 +29,12 @@ task('clean:docs', (cb) => {
 // ----------------------------------------
 
 task('build:docs:docgen', () => src([
-  `${config.paths.src()}/addons/**/*.js`,
-  `${config.paths.src()}/behaviors/**/*.js`,
-  `${config.paths.src()}/elements/**/*.js`,
-  `${config.paths.src()}/collections/**/*.js`,
-  `${config.paths.src()}/modules/**/*.js`,
-  `${config.paths.src()}/views/**/*.js`,
+  `${config.paths.src()}/addons/*/*.js`,
```

Comment:
I think that we need to store component specific functionality in its directory, this will allow to manage easier in future, see #1443.

Reply:
Yep, looks like we already have a flat structure any how.